src/json.cpp

```diff
@@ -108,7 +108,7 @@ void JsonObject::mark_visited( const std::string &name ) const
 void JsonObject::report_unvisited() const
 {
 #ifndef CATA_IN_TOOL
-    if( test_mode && report_unvisited_members && !reported_unvisited_members &&
+    if( report_unvisited_members && !reported_unvisited_members &&
```

Comment:
Not a big issue, but these two boolean variables have very similar names so it's easy to confuse them.

Reply:
I'd prefer not to change it (because I'm lazy and it avoids churn), but not strongly.